src/data/text/item_descriptions.h

```diff
@@ -2406,9 +2406,15 @@ static const u8 sWhiteHerbDesc[] = _(
     "lowered stat.");
 
 static const u8 sExpShareDesc[] = _(
+#if I_EXP_SHARE_ITEM >= GEN_6
+    "This device gives\n"
+    "exp. to other\n"
+    "party members.");
```

Comment:
How about
"Device that gives\n"
"all party members\n"
"Exp. points from battles."

Reply:
That last line has 25 characters, it'll never fit. The cap is about 18 characters depending on the 18th character used iirc :eyes:

Reply:
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/77138753/c5033a1a-1fab-4652-bbd4-724bbe91e0d6)

Reply:
Use this to test out window widths:
https://docs.google.com/spreadsheets/d/1ddoAyFxiOY4c792pM1Gr6VNt3--rTuG9OktE4K_aMFE/edit#gid=466352472